app/src/main/java/com/sopt/now/presentation/main/search/SearchAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.sopt.now.presentation.main.search
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import com.sopt.now.databinding.ItemReqresUserBinding
+import com.sopt.now.presentation.model.ReqresUser
+import com.sopt.now.util.DiffUtilCallBack
+
+class SearchAdapter : ListAdapter<ReqresUser, SearchViewHolder>(diffUtil) {
+
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SearchViewHolder {
+        return SearchViewHolder(
+            ItemReqresUserBinding.inflate(
+                LayoutInflater.from(parent.context),
+                parent,
+                false
+            )
+        )
+    }
+
+    override fun onBindViewHolder(holder: SearchViewHolder, position: Int) {
+        holder.onBind(currentList[position])
+    }
+
+    companion object {
+        val diffUtil = DiffUtilCallBack<ReqresUser>(
+            onContentsTheSame = { old, new -> old.name == new.name },
+            onItemsTheSame = { old, new -> old.name == new.name }
+        )
```

Comment:
두 로직에 차이가 있나요?

Reply:
```suggestion
       onContentsTheSame = { old, new -> old.name == new.name },
       onItemsTheSame = { old, new -> old == new }
```
리사이클러뷰 아이템 추가,삭제가 잘 안되었었는데 요렇게 고쳐야 겠네요..